ion-schema/src/isl/isl_constraint.rs

```diff
@@ -272,13 +272,14 @@ pub mod v_2_0 {
     }
 
     /// Creates a `fields` constraint using the field names and [IslVariablyOccurringTypeRef]s referenced inside it
-    pub fn fields<I>(fields: I) -> IslConstraint
+    /// and specify if the `fields` are closed or not (i.e. indicates whether only fields that are explicitly specified should be allowed or not).
+    pub fn fields<I>(fields: I, content_closed: bool) -> IslConstraint
```

Comment:
I prefer not to have such a strong association with `content: closed`.

```suggestion
    pub fn fields<I>(fields: I, is_closed: bool) -> IslConstraint
```